cdap-app-fabric/src/main/java/io/cdap/cdap/internal/app/runtime/monitor/RuntimeMonitors.java

```diff
@@ -159,4 +151,21 @@ public static void setupMonitoring(Injector injector, ProgramOptions programOpts
   private RuntimeMonitors() {
     // no-op
   }
+
+  /**
+   * Returns a module which defines remote authenticator override bindings if runtime monitoring type is URL, otherwise
+   * returns a null provider.
+   */
+  public static Module getRuntimeMonitorRemoteAuthenticatorModule(RuntimeMonitorType runtimeMonitorType,
```

Comment:
This method name is unnecessarily long. It is already inside the RuntimeMonitors class, so no need to repeat it in the method. Simply call it `getRemoteAuthenticatorModule`

Reply:
Done